### ISSUE TITLE
Grouped priority

### ISSUE THREAD

**Issue by slasktrat** [frodeheg/no.sparegris#120]
Instead of requiring every individual device to have a unique priority it would be very useful to be able to group some devices by priority somehow. So that each time a device needs to be turned off, the app should pick a random device in the lowest available group.

**Comment by frodeheg** [comment]
Yeah, some improvements should be made to the prioritization logic, not sure how this should best be presented in the UI yet but I guess I will get some ideas in the new year.

**Comment by slasktrat** [comment]
One simple approach UI-wize could be to replace the up/down arrows with a numeric input field. Either by default, or enabled by a checkbox for "advanced mode" or something. Another alternative could be a button to add separators in the list and then all devices between two separators should share priority.

**Comment by frodeheg** [comment]
Default everything to the same priority add new priorities on demand? Or default to a few groups? Like water heaters in one group (default high priority night, low priority else). Heaters in one group (high priority day, low priority night). AC-devices in one group (higher priority than heaters to avoid excessive switching) ?

**Comment by slasktrat** [comment]
I would say that the priority behaviour of existing users should be left unchanged/all be in the same priority group. But for new users, the defaults you list makes good sense.

**Comment by frodeheg** [comment]
Good point.
